src/lib/types.ts

```diff
@@ -10,3 +11,5 @@ export type Config = {
   models: Partial<Record<Prisma.ModelName, ModelConfig | boolean>>;
   defaultConfig?: ModelConfig;
 };
+//extended like this so we didnt have to make a PR directly to prisma. NestedAction dont contains groupBy
+export type NestedCustomParams = Omit<NestedParams, "action"> & {action: NestedAction | "groupBy"}
```

Comment:
`NestedCustomParams` should no longer be needed 👍

Reply:
great